inc/cleantalk-public-validate-skip-functions.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+use Cleantalk\Variables\Server;
+use Cleantalk\Variables\Post;
+
+/**
+ * Function for skip in ct_contact_form_validate_postdata()
+ * @return bool
+ */
+function skip_for_ct_contact_form_validate_postdata()
```

Comment:
Can we use more friendly names? Function name should reflect what it does. FE: if ( ct_contact_form_validate_postdata_skipped() ){}